mne/coreg.py

```diff
@@ -39,6 +39,7 @@
 fid_fname = pformat(bem_fname, name='fiducials')
 fid_fname_general = os.path.join(bem_dirname, "{head}-fiducials.fif")
 src_fname = os.path.join(bem_dirname, '{subject}-{spacing}-src.fif')
+high_res_head_fname = os.path.join(subject_dirname, 'surf', 'lh.seghead')
```

Comment:
see what surface is produced by 

mne make_scalp_surfaces

it's not the surface that should default to if you assume you ran "mne make_scalp_surfaces"